probe-rs/src/architecture/arm/communication_interface.rs

```diff
@@ -542,9 +540,9 @@ pub struct ArmChipInfo {
 
 impl ArmChipInfo {
     pub fn read_from_rom_table(
-        interface: &mut ArmCommunicationInterface,
+        mut interface: ArmCommunicationInterface,
```

Comment:
Why is this changed? A reference to the interface should still be enough, I think.